tests/vnc_test.go

```diff
@@ -155,6 +160,49 @@ var _ = Describe("[rfe_id:127][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			_, err = wrappedRoundTripper.RoundTrip(req)
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		It("should connect to vnc with --proxy-only flag", func() {
+
+			By("Invoking virtctl vnc with --proxy-only")
+			proxyOnlyCommand := tests.NewVirtctlCommand("vnc", "--proxy-only", "--namespace", vmi.Namespace, vmi.Name)
+
+			r, w, _ := os.Pipe()
+
+			// Stdout should be set back to previous value after the test
+			oldStdout := os.Stdout
+			os.Stdout = w
+			defer w.Close()
+			defer func() { os.Stdout = oldStdout }()
+
+			// Run this as go routine to keep proxy open in the background
+			go func() {
+				defer GinkgoRecover()
+				Expect(proxyOnlyCommand.Execute()).ToNot(HaveOccurred())
+			}()
+
+			var result map[string]interface{}
+			Eventually(func() error {
+				return json.NewDecoder(r).Decode(&result)
+			}, 60*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
```

Comment:
The timeout and the polling interval should be the same, it does not make sense to call the method more than once.

Reply:
Corrected, in this case we only need the timeout